Documentation/ceph-filesystem-crd.md

```diff
@@ -40,7 +40,7 @@ spec:
     - failureDomain: host
       replicated:
         size: 3
-  preservePoolsOnDelete: true
+  preserveFilesystemOnDelete: true
```

Comment:
Wrong rebase? This commit shouldn't be there...

Reply:
Yup, realized that... Corrected it...